src/DSi.cpp

```diff
@@ -185,6 +186,29 @@ void Reset()
     ARM7Write16(eaddr+0x42, 0x0001);
 }
 
+void SetupDirectBoot()
+{
+    // If loading a NDS directly, this value should be set
+    // depending on the console type in the header at offset 12h
+    if (NDS::IsDirectBoot)
```

Comment:
the if and the variable is unnecessary

Reply:
It is indeed DirectBoot when we are in the function (And thus to be removed)
However to the Variable, please see the comment at the LoadNAND function.